src/core/raster/qgscolorrampshader.h

```diff
@@ -130,7 +130,7 @@ class CORE_EXPORT QgsColorRampShader : public QgsRasterShaderFunction
     /** Maximum size of the color cache. The color cache could eat a ton of
      * memory if you have 32-bit data
      * @deprecated will be removed in QGIS 3.0 */
```

Comment:
Yes, I removed the Q_DECL_DEPRECATED because it caused compiler warnings, because it's used in it get/set methods. other option would be to remove this variable and replace get/set methods by stubs, but I ought it to be of too minor importance to think too much about it, as this function/variable will probably never be used in its existence.